gating/pre_merge_test/run

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
add -xeu

Reply:
maybe dump full env to the console as well? `env`

Reply:
Will doing that record any secrets as an artifact?